tests/installation/boot_into_snapshot.pm

```diff
@@ -38,6 +38,11 @@ sub run() {
             die "OS_VERSION after Rollback matches OS_VERSION before Rollback";
         }
     }
+    if (get_var('CHECK_SNAPPER_ROLLBACK_CLEANUP')) {
+        assert_script_run('snapper rollback');
+        assert_script_run('snapper --iso list');
+        assert_screen('snapper-autocleanup');
```

Comment:
while this works now I wonder if this is very robust or necessary to check with needles. why not just call

```
assert_script_run('snapper --iso list | grep \'number.*rollback\'', fail_message => 'rollback backup should be marked for cleanup');
```
?

This will also make explicit what you expect. Also you don't need two needles, one for SLE and one for openSUSE